website/project/model.py

```diff
@@ -215,84 +236,157 @@ def get_content(self, auth):
         return self.content
 
     @classmethod
-    def find_unread(cls, user, node):
+    def find_unread(cls, user, node, page=None, root_id=None, check=False):
```

Comment:
This might be better if it had count or something in the name. It's not obvious that it's finding the number of comments or the actual unread comments.